packages/@glimmer/compiler/lib/passes/1-normalization/keywords/impl.ts

```diff
@@ -115,20 +115,9 @@ export type KeywordNode =
   | ASTv2.InvokeBlock
   | ASTv2.ElementModifier;
 
-export function keyword<
```

Comment:
This code is only used in one place and abstracts very little, and the bulk of the implementation is
type annotations. ✂️